### ISSUE TITLE
Training Parameter Settings

### ISSUE THREAD

**Issue by snownus** [weiliu89/caffe#589]
The VOC model released solver.prototxt has parameters:
base_lr: 0.001
snapshot: 80000
stepvalue: 80000
stepvalue: 100000
stepvalue: 120000

Does your training process use those parameter values?

**Comment by youngwanLEE** [comment]
@snownus  Yes

**Comment by zgplvyou** [comment]
yes,but i had never got  the loss below 2 when training.the other people got the last loss in the range of 2 to 1.5

**Comment by weiliu89** [comment]
You may need to tune the parameters a bit.

Besides, the loss value itself is not very meaningful. It depends on how you normalize the loss. For example, faster rcnn normalize the loss by the total number of samples (pos + neg), so that the loss is small. SSD normalize the loss by the number of positive samples, so that the loss is big (roughly 4x bigger since pos : neg ratio is 1 : 3). Eventually how to get the best performance needs some tuning of the hyper parameters such as initial learning rate, etc.

**Comment by ryusaeba** [comment]
@weiliu89 I tried different initial learning rate and different learning rate decay combination and it seems loss stop around ~3. Could you please give me some idea how can I do to further reduce loss and improve accuracy. Another quick question when you achieve 0.78 mAP on testing set, how about your mAP on training set?

**Comment by ujsyehao** [comment]
Hi @ryusaeba, I have problems with learning rate, 
`python examples/ssd/ssd_pascal.py`
author use finetune in this script, so it sets base_lr to 0.001, but as for add_extra_layers, why not set lr_mult to 10 or bigger?
